public/stylesheets/itvc.css

```diff
@@ -326,8 +326,11 @@ html[data-useragent*='MSIE 10.0'] .flex-container {
     }
 }
 
-
-
-
+#global-header .header-proposition a.menu {
+    display: none;
+}
+.js-enabled #global-header .header-proposition #proposition-links {
+    display: block;
+}
```

Comment:
Could you add a comment into the CSS to explain what this is for?
Will help in the future if we can ever remove our custom css to know what it was for

Reply:
fixed.